java/vector/src/test/java/org/apache/arrow/vector/TestListVector.java

```diff
@@ -92,6 +93,26 @@ public void testCopyFrom() throws Exception {
       Assert.assertFalse("should be null", reader.isSet());
       reader.setPosition(2);
       Assert.assertTrue("shouldn't be null", reader.isSet());
+
+      /* check the exact contents of vector */
+      final ListVector.Accessor accessor = outVector.getAccessor();
+
+      /* index 0 */
+      Object result = accessor.getObject(0);
+      ArrayList<Long> resultSet = (ArrayList<Long>)result;
```

Comment:
Actually, [google_checks.xml](https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/google_checks.xml) doesn't specify [TypecastParenPad](http://checkstyle.sourceforge.net/config_whitespace.html#TypecastParenPad) rule, so this is fine.